### PR TITLE
Feature light mode design

### PR DESCRIPTION
**Pull request by j1mb0b**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,4 @@
 node_modules
 /dist/orejime.js
-/dist/orejime.css
-/dist/orejime.scss
 /es
```
Comment (j1mb0b) [inline .gitignore]: @laurent-b4y this should not be changed

File: dist/orejime.scss [hunk 2]
```diff
@@ -0,0 +1,432 @@
+$orejime-box-shadow: 0 4px 6px 0 rgba(0, 0, 0, 0.2), 5px 5px 10px 0 rgba(0, 0, 0, 0.19) !default;
```
Comment (j1mb0b) [inline dist/orejime.scss]: @laurent-b4y We should not change and add dist files (that get generated by the compiler). Instead, can you make your changes in the src dir ?

Comment (j1mb0b) [reply dist/orejime.scss]: Here is the location: src/scss/orejime.scss

**Review by laurent-b4y — COMMENTED** [review]
Looks good

**Review by laurent-b4y — APPROVED** [review]

**Pull request merged**